### DISCUSSION TITLE
How to change the default interaction hand/what is it's animation?

### DISCUSSION THREAD

**Discussion by core-hacked** [google/model-viewer#4640] [Q&A]
Hi, I have read the docs about the interaction and can create a custom prompt with my own image. But what is the default animation values?

Keyframes from [the example](https://modelviewer.dev/examples/stagingandcameras/#customPrompt).
```js
const finger0 = {
          x: {
            initialValue: 0.6,
            keyframes: [
              { frames: 1, value: 0.7 },
              { frames: 1, value: 0.5 },
              { frames: 1, value: 0.7 },
              { frames: 1, value: 0.6 },
            ],
          },
          y: {
            initialValue: 0.45,
            keyframes: [
              { frames: 1, value: 0.4 },
              { frames: 1, value: 0.3 },
              { frames: 1, value: 0.4 },
              { frames: 1, value: 0.45 },
            ],
          },
        };
```

Or is it possible to just change the hand without the custom interaction code?

**Comment by elalish** [comment]
Yes, you can just change the hand using our slot: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-slots-interactionPrompt

**Reply by core-hacked** [reply]
Hi, I tried that, but the icon seems to be the stock one.. That's why I asked. I remade it somewhat with the custom interaction but eh, maybe a bug? I will update this with more info tomorrow. No rush
